Fixing errors found after IESG approval

italobusi opened the pull request:
Fixing must statement for max-slot-width-factor  (fix #16 )

italobusi committed:
Update ietf-layer0-types.yang

Fixing must statement for max-slot-width-factor  (Issue #16 )

haomianzheng reviewed (approved):
This change has been checked via IETF online validator.

italobusi committed:
Update ietf-layer0-types.yang

Fixed copyright date

agva123 reviewed (approved):
Yes it needs to be fixed.

italobusi committed:
Fix #18

Fixing editorial issue with the definition of the anchor wavelength

italobusi commented:
@haomianzheng : check with RFC Editor before merging this PR

italobusi committed:
Update ietf-layer0-types.yang

Fix only issue #16

haomianzheng committed:
Update ietf-layer0-types.yang

Integrate comments from AUTH48.

haomianzheng committed:
Update ietf-layer0-types.yang

Update according to https://www.rfc-editor.org/authors/ietf-layer0-types@2021-07-14-yang-rfcdiff.html, from the RFC editor.

haomianzheng committed:
Update ietf-layer0-types.yang

change the email address for Victor.

haomianzheng committed:
Update ietf-layer0-types.yang

italobusi committed:
Update ietf-layer0-types.yang

Changed C band with DWDM grid

italobusi committed:
Update ietf-layer0-types.yang

Aligned with RFC Editors changes

italobusi committed:
Update ietf-layer0-types.yang

Changed C band with DWDM grid

italobusi committed:
Merge branch 'rfc-process' of https://github.com/ietf-ccamp-wg/draft-ietf-ccamp-layer0-types into rfc-process
